seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumRun.scala

```diff
@@ -171,18 +163,31 @@ private[selenium] object SeleniumRun {
   private def maybeCleanupDriver(d: WebDriver, config: SeleniumJSEnv.Config) =
     if (!config.keepAlive) d.close()
 
-  private def htmlPage(scripts: Seq[java.net.URL]): String = {
-    val scriptTags =
-      scripts.map(path => s"<script src='${path.toString}'></script>")
+  private def htmlPage(setupJsPath: Path, input: Seq[Input], materializer: FileMaterializer): String = {
+    val setupJs = makeTag(setupJsPath, "text/javascript", materializer)
```

Comment:
Hum, am I misunderstanding this now, or would this copy the setup script twice?

If you are looking for ways to consolidate code and the fact that the setup script is not a `Path` is problematic, I think it would be OK to add Jimfs as a dependency and make in-memory `Path`s. There is precedent for this in the Node.js env: https://github.com/scala-js/scala-js-js-envs/blob/v1.1.1/nodejs-env/src/main/scala/org/scalajs/jsenv/nodejs/ComSupport.scala#L247

Reply:
Done. Let me know if I got it right.